### PR TITLE
chore: Format css files

### PR DESCRIPTION
**Pull request by marcustyphoon**
### Description

This uses prettier to format our CSS files.

I don't love every diff here, but they're mostly reasonable.

I was originally going to use biomejs for this (this was actually the result of trying it out), but it handled a few edge cases kind of poorly.

### Testing steps

I didn't test this.

### CODE REVIEW

File: src/action/backup.css [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
   background-color: rgb(var(--passive-grey));
   color: rgb(var(--black));
-  font-family: monospace, monospace;
```
Comment (marcustyphoon) [inline src/action/backup.css]: I don't think there's a reason for this... right?

Comment (marcustyphoon) [reply src/action/backup.css]: Aha: https://github.com/necolas/normalize.css/issues/519#issuecomment-197131966

**Review by AprilSylph — APPROVED** [review]
